haystack/errors.py

```diff
@@ -203,3 +203,20 @@ class HuggingFaceInferenceUnauthorizedError(HuggingFaceInferenceError):
 
     def __init__(self, message: Optional[str] = None, send_message_in_event: bool = False):
         super().__init__(message=message, status_code=401, send_message_in_event=send_message_in_event)
+
+
+class CohereInferenceError(NodeError):
```

Comment:
What's the difference between `CohereInferenceError` and [`CohereError`](https://github.com/deepset-ai/haystack/blob/e6c228e2f5ce83274778c6727341d729d5ff5ba6/haystack/errors.py#L160)?

Reply:
None, good catch, let me update the PR.